app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/terminationsuccess/TerminationSuccessDestination.kt

```diff
@@ -35,7 +35,7 @@ import kotlinx.datetime.toJavaLocalDate
 internal fun TerminationSuccessDestination(
   terminationDate: LocalDate?,
   onSurveyClicked: () -> Unit,
-  navigateToInsurances: () -> Unit,
```

Comment:
Cool! my only concern here is that we go back, but not to the insurances, I mentioned it here: https://hedviginsurance.slack.com/archives/C03HT2JRDPG/p1712163628112979?thread_ts=1712150517.177999&cid=C03HT2JRDPG